### PR TITLE
Bugfix/Fix python programming exercise template issues

### PR DESCRIPTION
**Pull request by thilo-behnke**
### Checklist
- [x] I tested the changes and all related features on the test server https://artemistest.ase.in.tum.de.
- [x] ~I documented my source code using the JavaDoc / JSDoc style.~
- [x] ~I added integration test cases for the server (Spring) related to the features~
- [x] ~I added integration test cases for the client (Jest) related to the features~
- [x] ~I added screenshots/screencast of my UI changes~
- [x] ~I translated all the newly inserted strings~

### Motivation and Context

Fixes issues from https://github.com/ls1intum/Artemis/issues/795.

### Description

For c & python:
- Repair template so build does not fail for template
- Remove tests that are always passing from repo and readme
- Adjust build plan settings for sequential test runs

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration > Create Programming Exercise > Language Python > Sequential Test Runs true or false
3. Go to edit in editor and check that the template result proper test feedback (see screenshot) with 13 failed tests

### Screenshots

image

### CODE REVIEW

File: src/main/java/de/tum/in/www1/artemis/service/connectors/BambooBuildPlanService.java [hunk 1]
```diff
@@ -117,9 +117,8 @@ else if ((programmingLanguage == ProgrammingLanguage.PYTHON || programmingLangua
         }
         else if (programmingLanguage == ProgrammingLanguage.PYTHON || programmingLanguage == ProgrammingLanguage.C) {
             return defaultStage.jobs(new Job("Default Job", new BambooKey("JOB1")).tasks(checkoutTask,
-                    new ScriptTask().description("Builds and tests the structural tests")
-                            .inlineBody("pytest tests/structural/* --junitxml=test-reports/structural-results.xml\nexit 0"),
-                    new ScriptTask().description("Builds and tests the behavior tests").inlineBody("pytest tests/behavior/* --junitxml=test-reports/behavior-results.xml\nexit 0"),
+                    new ScriptTask().description("Builds and tests the structural tests").inlineBody("pytest structural/* --junitxml=test-reports/structural-results.xml\nexit 0"),
```
Comment (thilo-behnke) [inline src/main/java/de/tum/in/www1/artemis/service/connectors/BambooBuildPlanService.java]: I only changed the folder structure in the pytest command here.

**Comment by lovaiible** [comment]
Hi @thilo-behnke, I just tested the bug fix and it worked fine for Python! Good job :)
However I discovered that it still faces some issues when creating programming exercises with C as programming language (see screenshot)? 
image
And also the files for C exercises, solution and test are all in Python? I am not familiar with this topic, so I'm a bit confused here. Maybe you can explain that to me? :)

**Comment by thilo-behnke** [comment]
> Hi @thilo-behnke, I just tested the bug fix and it worked fine for Python! Good job :)
> However I discovered that it still faces some issues when creating programming exercises with C as programming language (see screenshot)?
> image
> And also the files for C exercises, solution and test are all in Python? I am not familiar with this topic, so I'm a bit confused here. Maybe you can explain that to me? :)

Thanks for your review and testing! You are right, I missed to remove the functions also from the c structural test file...

**Comment by ungaralex** [comment]
> Hi @thilo-behnke, I just tested the bug fix and it worked fine for Python! Good job :)
> However I discovered that it still faces some issues when creating programming exercises with C as programming language (see screenshot)?
> image
> And also the files for C exercises, solution and test are all in Python? I am not familiar with this topic, so I'm a bit confused here. Maybe you can explain that to me? :)

The C programming courses want to use pytest/general Python tests for their exercises, since there seems to be no proper unit testing framework in C. At least not like the Python unit tests. That's why we are essentially using the same template here :)

**Comment by krusche** [comment]
@jpbernius in the last PR that introduced the regression, we basically wanted to have the same checkout structure for all programming languages that support the automatic setup.
The test repository is checked out in the root repository.
The assignment repository is checked out in the folder „assignment“.

As far as I can see this is still the case in this PR and it only fixes the structure of the test script that executes the test cases, so this should be fine for me!

**Pull request merged**